examples/sveltekit-scoped/package.json

```diff
@@ -14,6 +14,7 @@
     "@sveltejs/kit": "next",
     "@unocss/core": "link:../../packages/core",
     "@unocss/preset-icons": "link:../../packages/preset-icons",
+    "@unocss/transformer-directives": "^0.45.26",
```

Comment:
Better use a link here

Reply:
I wanted to do that but I don't understand the error I receive when I do:

```
failed to load config from C:\dev\unocss\unocss\examples\sveltekit-scoped\vite.config.ts     
error when starting dev server:
Error: TRANSFORM_ERROR: C:\: Export namespace should be first transformed by `@babel/plugin-proposal-export-namespace-from`.
C:/dev/unocss/unocss/node_modules/.pnpm/css-tree@2.2.1/node_modules/css-tree/lib/index.js:0:0    
at C:/dev/unocss/unocss/packages/transformer-directives/src/index.ts:4:16
at jiti (C:\dev\unocss\unocss\node_modules\.pnpm\jiti@1.15.0\node_modules\jiti\dist\jiti.js:1:247497)    
at file:///C:/dev/unocss/unocss/packages/transformer-directives/dist/index.mjs:4:71      
at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
at async Promise.all (index 0)    
at async ESMLoader.import (node:internal/modules/esm/loader:533:24)    
at async loadConfigFromBundledFile (file:///C:/dev/unocss/unocss/examples/sveltekit-scoped/node_modules/.pnpm/vite@3.1.6/node_modules/vite/dist/node/chunks/dep-db16f19c.js:63505:21)     
at async loadConfigFromFile (file:///C:/dev/unocss/unocss/examples/sveltekit-scoped/node_modules/.pnpm/vite@3.1.6/node_modules/vite/dist/node/chunks/dep-db16f19c.js:63400:28)            
at async resolveConfig (file:///C:/dev/unocss/unocss/examples/sveltekit-scoped/node_modules/.pnpm/vite@3.1.6/node_modules/vite/dist/node/chunks/dep-db16f19c.js:63017:28)    
at async createServer (file:///C:/dev/unocss/unocss/examples/sveltekit-scoped/node_modules/.pnpm/vite@3.1.6/node_modules/vite/dist/node/chunks/dep-db16f19c.js:62083:20)
```

Reply:
Just discovered that I don't need to add that to the package.json at all because it's already in core. So I removed it.